testing/test_all.py

```diff
@@ -4,7 +4,15 @@
 
 from testing.yuca.test_multiverse import TestCA
 from testing.yuca.test_activations import TestGaussian, TestDoGaussian
-from testing.yuca.test_utils import TestQueryKwargs
+from testing.yuca.test_utils import TestQueryKwargs,\
+        TestSeedAll, \
+        TestGetMask, \
+        TestGetBiteMask,\
+        TestGetAperture,\
+        TestPrepInput, \
+        TestMakeTarget, \
+        TestPlotGridNbhd,\
+        TestPlotKernelGrowth
```

Comment:
The remaining untested functions in `utils` have to do with writing gif animations.